liquidprompt

```diff
@@ -2691,6 +2733,12 @@ _lp_default_theme_prompt_data() {
         LP_VENV=
     fi
 
+    if _lp_container_color; then
+        LP_CONTAINER=" $lp_container_color"
+    else
+        LP_CONTAINER=
```

Comment:
Adding a template section needs to be added to the template section docs:
https://liquidprompt.readthedocs.io/en/v2.0.0-rc.1/theme/default.html#template-sections

Reply:
Would it be possible for you to generate a screenshot and update the alt tag?

Reply:
Would for sure be possible, but nothing would change without work. The screenshots are generated with `tools/theme-preview.sh`, which manipulates which features are enabled, and mocks the data sources to return the exact same data each time.

I'm not against adding things to this script as features are added, but showing _everything_ that Liquidprompt supports would probably be counterproductive. Especially since this is essentially another development environment, and will look very similar to the others Liquidprompt supports, one of which (Python venv) is part of the theme preview generation already.

So my thought right now is that this shouldn't get added.

Reply:
Ahh.  I mentally elided the word "nearly" from "displays nearly everything" in the documentation.  Nevermind!  ;-)